templates/cicd/pipeline_cfn.yml

```diff
@@ -234,7 +234,7 @@ Resources:
               - Name: SCCheckoutArtifact
             OutputArtifacts:
               - Name: BuildOutput
-{{$length := len .Stages}}{{if gt $length 0}}{{range $stage := .Stages}}
+{{$length := len .Stages}}{{if gt $length 0}}{{range $stage := .Stages}}{{$appNum := len $stage.LocalApplications}}{{if gt $appNum 0}}
```

Comment:
Could you add an unit test for the case where there's no app?

Reply:
Addressed